core/src/main/scala/io/finch/Module.scala

```diff
@@ -0,0 +1,15 @@
+package io.finch
+
+import cats.effect.Effect
+import io.finch.syntax.EndpointMappers
+
+abstract class Module[F[_]](implicit val E: Effect[F]) extends Endpoints[F]
+  with Outputs
+  with ValidationRules
+  with EffectInstances[F] {
+
+  type Endpoint[A] = io.finch.Endpoint[F, A]
+
+  object syntax extends EndpointMappers[F]
```

Comment:
One more thing. I believe we can get rid of `syntax` now - let's just mix it with other things.